packages/hawtio/src/plugins/camel/routes-service.ts

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import { Logger } from '@hawtiosrc/core'
+import { MBeanNode } from '@hawtiosrc/plugins/shared'
+import { jolokiaService } from '@hawtiosrc/plugins/connect/jolokia-service'
+import { jmxDomain } from './globals'
+import $ from 'jquery'
+import { schemaService } from './schema-service'
+import * as icons from './icons'
+
+const log = Logger.get(`hawtio-${jmxDomain}-routes-service`)
```

Comment:
I think it's too much for logging. Here we can just use the global logger named 'hawtio-camel' from `./globals`. For starters, I'd prefer to keep the same level of logger granularity as the legacy hawtio-integration does.